big_tests/tests/carboncopy_SUITE.erl

```diff
@@ -250,7 +248,7 @@ carbons_get_enabled(ClientOrClients) ->
 
 given_client_logs_out(Config, Client) ->
     escalus_client:stop(Config, Client),
-    timer:sleep(300).
+    timer:sleep(100).
```

Comment:
Can't `wait_until` be used here?

Reply:
In such cases waiting until user's session disappear from the server should be the most stable solution, in my opinion. Sth what we already have here: https://github.com/esl/MongooseIM/blob/c3d2ffe66b297560f2da03cd53cb37c9c7dddc5b/big_tests/tests/push_helper.erl#L77-L84